src/Sulu/Bundle/HttpCacheBundle/Cache/CacheManager.php

```diff
@@ -79,4 +79,9 @@ public function supportsInvalidate(): bool
     {
         return $this->fosCacheManager->supports(FOSCacheManager::INVALIDATE);
     }
+
+    public function supportsInvalidateTag(): bool
```

Comment:
why break from the `FOSCacheManager` naming?

```suggestion
    public function supportsTags(): bool
```